public/demo-html-input/assets/bar-chart-interaction.html

```diff
@@ -5,7 +5,7 @@
     <title>D3 Nice Axes</title>
     <script src="https://d3js.org/d3.v7.js"></script>
     <!-- Load revisit-communicate to be able to send data to reVISit -->
-    <script src="../js/revisit-communicate.js"></script>
+    <script src="../../revisitUtilities/revisit-communicate.js"></script>

```

Comment:
For things like this, can we provide an absolute path, instead of relative? I'm not sure this would work, but something like
`src="/study/revisitUtilities/revisit-communicate.js"`